### PR TITLE
Add multiplication

### PR DESCRIPTION
**Pull request by jerivasuaq**
Try to implement the multiplication in a function, but it doesnt work

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -13,32 +13,25 @@ def main():
...
-        print(f"The resultant matrix is:")
-        for i in range(A.r):
-            for j in range(B.c):
-                print(f"{C.matrix[i][j]}\t", end="")
-            print()
-
-    else:
-        print("The given matrices cannot be multiplied!")
-        print(f"Matrix A has {A.c} columns and Matrix B has {B.r} rows")
+
```
Comment (Fractalemma) [inline main.py]: This code fragment is not useful, can you remove it?

Comment (CaballeroTP) [inline main.py]: This lines are not necessary for the project

File: matrix.py [hunk 2]
```diff
@@ -25,5 +25,17 @@ def print(self):
                 print(f"{self.matrix[i][j]}\t", end="")
             print()
 
-    def mul(self):
-        pass
\ No newline at end of file
+    def mul(self, B):
```
Comment (CaballeroTP) [inline matrix.py:28]: I like this implementation from project. :)

**Review by Fractalemma — CHANGES_REQUESTED** [review]

**Review by CaballeroTP — CHANGES_REQUESTED** [review]

File: main.py (hunk 1 above)
Comment (jerivasuaq) [reply main.py]: Solved!

**Review by Fractalemma — APPROVED** [review]
LGTM

**Review by CaballeroTP — APPROVED** [review]
LGTM! :) 
Good Job  GD

**Pull request merged**